test/suite/Data/HashTable/Test/Common.hs

```diff
@@ -136,7 +136,7 @@ testInsert prefix dummyArg =
     prop :: GenIO -> ([(Int, Int)], (Int,Int)) -> PropertyM IO ()
     prop rng o@(origL, (k,v)) = do
         announceQ "insert" o
-        let l = V.toList $ shuffle rng $ V.fromList $ remove k $ dedupe origL
+        let l = V.toList $ shuffleVector rng $ V.fromList $ remove k $ dedupe origL
```

Comment:
I'll commit this for now but this line is long and we try to stay within 80col -- could you please refold this line in a subsequent changeset?